CMakeLists.txt

```diff
@@ -130,11 +130,11 @@ endif()
 IF(MSVC)
 	SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /EHsc")
 ELSEIF(APPLE)
-	if (CMAKE_CXX_COMPILER_ID MATCHES "Clang")
-		SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fobjc-arc -ObjC++")
-	ELSE()
+	#if (CMAKE_CXX_COMPILER_ID MATCHES "Clang")
```

Comment:
commenting this out breaks fixed issue https://github.com/SaschaWillems/Vulkan/pull/770
revert?

Reply:
The issue here is the kind of cmake generator used for macOS builds.  The comment-out was to meant to support Xcode builds.  However, you are right that it appears to break macOS builds when using cmake's make or ninja command-line generators.  I will fix this by testing for CMAKE_GENERATOR and selecting the correct objective c++ option.  Note the previous test for "Clang" did not make sense since it has been the default macOS compiler for ages and is used for both Xcode and command line builds.  I will link a pull request here when ready.

Reply:
Changes implemented in PR #1117.  Please let me know if this fixes your issue.